docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -573,10 +563,6 @@ The following properties are available for use:
     - Set the checkpoint interval in seconds.
   * - ``change_data_feed_enabled``
     - Enables storing change data feed entries.
-  * - ``reader_version``
-    - Set reader version.
-  * - ``writer_version``
-    - Set writer version.
 
 The following example uses all six table properties::
```

Comment:
`six` -> `four`

Reply:
good catch. very tricky

"The following example uses multiple table properties" would avoid having to update that line.